ibm/resource_ibm_iam_trusted_profile.go

```diff
@@ -0,0 +1,313 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/platform-services-go-sdk/iamidentityv1"
+)
+
+func resourceIBMIamTrustedProfile() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMIamTrustedProfileCreate,
+		ReadContext:   resourceIBMIamTrustedProfileRead,
+		UpdateContext: resourceIBMIamTrustedProfileUpdate,
+		DeleteContext: resourceIBMIamTrustedProfileDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Name of the trusted profile. The name is checked for uniqueness. Therefore trusted profiles with the same names can not exist in the same account.",
+			},
+			"account_id": &schema.Schema{
```

Comment:
Do we need account_id can't we set the account_id internal 
```
userDetails, err := meta.(ClientSession).BluemixUserDetails()
	if err != nil {
		return err
	}
	```
	Can we use this and set the accountID internally similar to cli